File: src/template/pedro/Resta.java
package template.pedro;

class Resta extends Operacion {
    Resta() {
    }

    Resta(int redondeo) {
        this.redondeo = redondeo;
    }

    void getOperacion() {
        System.out.println("operación: resta");
    }

    void calcula() {
        resultado = primerValor - segundoValor;
    }
}

File: manuel/template/Resta.php
<?php 
	require_once "Operacion.php";
	
	class Resta extends Operacion{
		public function calcula()
		{
			$this->resultado = $this->valor1 - $this->valor2;
		}

		public function getOperacion()
		{
			echo 'Resta <br/>';
		}
	}
?>